br/pkg/backup/client.go

```diff
@@ -69,6 +71,13 @@ type Checksum struct {
 // ProgressUnit represents the unit of progress.
 type ProgressUnit string
 
+const (
+	// RangeUnit represents the progress updated counter when a range finished.
+	RangeUnit ProgressUnit = "range"
```

Comment:
Maybe `Unit{Range|Region}`. Use the same prefix will be more friendly to autocomplete.